cmd/app/app.go

```diff
@@ -77,6 +82,23 @@ func (a *App) Run() error {
 	return ui.StartUILoop()
 }
 
+func (a *App) setMaxProcs() {
+	if a.Flags.MaxCores >= 1 {
+		maxProcs := a.Flags.MaxCores
+
+		// Limit GOMAXPROCS(n) to number of CPU's
+		if maxProcs > runtime.NumCPU() {
+			maxProcs = runtime.NumCPU()
```

Comment:
This line still has no effect, because of the return on the next line.

Reply:
Yes exactly! 
I should just make a return, without unnecessary actions, because the variable GOMAXPROCS is set to NumCPU by default, and also I may not use additional variables.

Next time I'll try to get enough sleep, before any coding =)